### PR TITLE
factor out sc_isolation_entity

### PR DESCRIPTION
**Pull request by mccalluc**
Use the suggest-edit mechanism to indicate which is the correct definition for this field; Chuck will followup and regenerate docs.

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# scrnaseq-hca-v0.yaml
+- name: sc_isolation_entity
+  constraints:
+    required: true
+  description: The type of single cell entity derived from isolation protocol
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml:5]:
```suggestion
  description: The type of single cell entity derived from isolation protocol
  constraints:
     enum:
         - whole cell
         - nucleus
         - cell-cell multimer
         - spatially encoded cell barcoding
```

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# scrnaseq-hca-v0.yaml
+- name: sc_isolation_entity
+  constraints:
+    required: true
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml:4]:
```suggestion
```

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml (hunk 1 above)
Comment (cebriggs7135) [reply src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml:5]: Add the same enums to this field for HCA as for the other sc/sn seq schema.

**Review by cebriggs7135 — CHANGES_REQUESTED** [review]
Make the same enums for this field in HCA schema as for the other sc schema.

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# scrnaseq-hca-v0.yaml
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml:1]:
```suggestion
```

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+# scatacseq-v0.yaml; scatacseq-v1.yaml
+- name: sc_isolation_entity
+  description: The type of single cell entity derived from isolation protocol.
+  constraints:
+    enum:
+      - whole cell
+      - nucleus
+      - cell-cell multimer
+      - spatially encoded cell barcoding
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_entity.yaml:29]:
```suggestion
```

**Pull request merged**